Content.Server/_RMC14/Rules/CMDistressSignalRuleSystem.cs

```diff
@@ -1779,6 +1786,15 @@ public void CancelPlanetVote()
 
     private string GetRandomOperationName()
     {
+
+        if (_usingCustomOperationName && OperationName != null)
+        {
+            _usingCustomOperationName = false;
+
+            return OperationName;
+        }
+        ReloadPrototypes();
```

Comment:
Is this intended to be here? It will just re-parse valid operation names which doesn't seem necessary